test/unit/query/token-visitors.js

```diff
@@ -0,0 +1,83 @@
+const Knex = require('../../../lib/index');
+const { expect } = require('chai');
+const { TokenListVisitor } = require('../../../lib/query/token-visitors');
+
+describe.only("Token visitor", () => {
```

Comment:
Probably ".only" wasn't intended to be committed :)

Reply:
Yes, removed.

Reply:
Yeah :) Removed.